src/mbgl/programs/uniforms.hpp

```diff
@@ -37,9 +37,14 @@ MBGL_DEFINE_UNIFORM_SCALAR(Size, u_texsize);
 MBGL_DEFINE_UNIFORM_SCALAR(bool, u_pitch_with_map);
 MBGL_DEFINE_UNIFORM_SCALAR(float, u_camera_to_center_distance);
 MBGL_DEFINE_UNIFORM_SCALAR(float, u_fade_change);
+MBGL_DEFINE_UNIFORM_SCALAR(float, u_weight);
 
 MBGL_DEFINE_UNIFORM_VECTOR(float, 2, u_extrude_scale);
 
+namespace heatmap {
+MBGL_DEFINE_UNIFORM_SCALAR(float, u_extrude_scale);
+} // namespace heatmap
+
```

Comment:
Other program-specific uniforms are defined in `xxx_program.hpp` -- should we move this one to `heatmap_program.hpp`?

Reply:
I would leave it here until we refactor https://github.com/mapbox/mapbox-gl-native/issues/11026 so that it's clear why we namespace it and that there's a collision.